### PR TITLE
Added support for device 008-304 (HiSense Window air conditioner)

### PR DESCRIPTION
**Pull request by flamingotter**
This is a hisense Window air conditioner. tested and working within my HA environment

### CODE REVIEW

File: custom_components/connectlife/data_dictionaries/008-304.yaml [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+      target: hvac_mode
+      options:
+        0: fan_only
+        2: cool
+        3: dry
+        5: eco
+  - property: t_beep
+    disable: true
+  - property: t_eco
+    disable: true
```
Comment (oyvindwe) [inline custom_components/connectlife/data_dictionaries/008-304.yaml:64]: Does the device expose any of the disabled properties?

**Comment by oyvindwe** [comment]
Thank you for your contribution ❤️ 

Just left a question in the review.

File: custom_components/connectlife/data_dictionaries/008-304.yaml (hunk 1 above)
Comment (flamingotter) [reply custom_components/connectlife/data_dictionaries/008-304.yaml:64]: Not that I was able to identify. This is a small room window AC unit with minimal features.

Comment (oyvindwe) [reply custom_components/connectlife/data_dictionaries/008-304.yaml:64]: So can the disabled properties just be deleted from the mapping file? Or are they included in the `statusList` for the device, but just don't have any effect?

Comment (flamingotter) [reply custom_components/connectlife/data_dictionaries/008-304.yaml:64]: can be deleted. I was just following how the other files were structured. I've commented them out of my local config and reloaded system without error.

Comment (flamingotter) [reply custom_components/connectlife/data_dictionaries/008-304.yaml]: commited change

**Comment by oyvindwe** [comment]
I think it's cleaner to just remove the properties, but I'm happy to merge in any case.

Thank you for your contribution!

**Pull request merged**